pkg/batchd/policy/drf/drf.go

```diff
@@ -64,7 +64,12 @@ func (drf *drfScheduler) Allocate(queues []*cache.QueueInfo, nodes []*cache.Node
 	for _, q := range dq {
 		psi := q.Value.(*podSetInfo)
 
-		assigned := drf.assignMinimalPods(psi.insufficientMinAvailable(), psi, nodes)
+		// fetch the nodes that match PodSet NodeSelector and NodeAffinity
+		// and store it for following DRF assignment
+		matchNodes := fetchMatchNodeForPodSet(psi, nodes)
+		psi.matchNodes = append(psi.matchNodes, matchNodes...)
```

Comment:
why add a `matchNodes` in psi?

Reply:
just store it for DRF assignment after minimal assignment, I will remove it from `psi` and use a map in `drf.go`